firmware/Core/Inc/eps_drivers/eps_debug_tools.h

```diff
@@ -0,0 +1,11 @@
+
+#ifndef __INCLUDE_GUARD__EPS_DEBUG_TOOLS_H__
+#define __INCLUDE_GUARD__EPS_DEBUG_TOOLS_H__
+
+#include "eps_drivers/eps_types.h"
+
+void EPS_debug_uart_print_system_status(EPS_result_system_status_t* system_status);
```

Comment:
How far should upper-case namespace extend? Like EPS_DEBUG_UART_print_system_status(...

Reply:
Just to match one of the namespaces listed in the [/docs/C_Namespaces.md file](https://github.com/CalgaryToSpace/CTS-SAT-1-OBC-Firmware/blob/main/docs/C_Namespaces.md). So in this case, just `EPS_xxxx` is good.

The goal with the namespaces is just to emphasize that every piece of code in repo needs to have a reason to be in the repo, and a reason to be executed/used. Namespaces help avoid accidentally using them between systems that don't make sense (especially in struct/enum typedefs, where there could otherwise be multiple `result_system_status_t` struct definitions).